.travis.yml

```diff
@@ -0,0 +1,40 @@
+    - secure: NTEJa1wL27hu9LrSNELh18Zzfdr5dr76wEYJz+E04gixJ3R3XyaU4B8MAodMvAC8V1du1BhP3pZHYby6ZxnSOhPZvAZL4FbEFkOPs1oS/qgpam8sf5+HYplfNZ+3vKxz0yxPdPE/Ltk1gsVqgOtjXyX4VTaCNtoH++pYAXBxdkjoQ1CwgmT6fIBGliH+Kf6b7z1Br+DDycaslMnlWu0guWNRwPKs/M8VPL3C3wEE6zEbAPtaSDYaB4fLqLwaKrjiOxKsKIlOV8A+BLZMPUmq3jlnrv6f3Ic2PNluiDnOVf1xzKHogt6KX0JBUANYpsE3qVDXMI76Ulv0S+446Pp1hxla3qj8noDPZal0xKG8S94aKEN01+QnktXbezK61/ErI9JCUk9AinjtT6uOk6MjLSjD0sLSwsNjvB3Ba4T4GEa0KI34wbH1PSdJJX5wyXHawxPK3eGwgqOvnUQtJbXGmgbhq4O0UUWbwOg/M7sf2Sb/85wZtwf5Ok134t9Uq0QJFyq+m+Hy8Yo+iKDAVXaok3ZQ7LIdC4EnnuFUOR6Kg3VIVu4zdiuLMmDUfR2pXhlS1+Ac9KGwuDePgBAW9lCKWm9r+AvheXAJ7CJASFSVyXJq+tWXjRutkrvaBpKzx2Z6iw0EWkVVl4bVPAGJozfhrRADNsD9yPZWY4ff3qqn34o=
+    - secure: gEbMsr7aBa7eBZA4vlYi5NxHJYnJHdyTlPCdLLE90+B9i8a6a0J1/U5nhy3GlW4DeufXQjCqzY4tDQNvvHOzsAwYQGtFXCkNptzlpL/dAqScRWq61bl7VO6Zplq3AX2dIlQJdr09rKQKL1QqBcxxOZeNGG32vdA0GuGnRJHxA7rr4M218UY/KPR0u5ijrPVll8QtAPx2vzx1+EoIQ677ppRYE9oDXfeu3c9NvRCR5Lk4Iq8hXBIRLtEZrVtKzoAIf/9SoOCn3gK0/yT4NuOn+EDkJzVSK1bS0GMLnBUR5HY27/uNfugwFgLFlVILzvceRDVTaFbov0+jT2B8gH4LM6M9b0K1Fhd/eaIcgSJ/gcKc+bbvaihoXySXtFmQaCxzo/qqgdY5g4kZamENfxMYeutl2Tanno7qsIsjs2Lz2UTh1mP03zIjCJu8rMMJCgf3Ql++fVtq98Rrgg2ZpLOdrwLB1RtJ7BaXS8ZCDJ4db8xOEFd8rwadClMpK7bCaRb5uLr5LX5jh7rHypGmf+m5kAdBmeyDvdw/zQ/u/8Q44acm+O0IaDMoE453pbAMY7hmt4aKOsQHM1sX5F8X44E9Y8HzQfb1ku/DTAsigPjaqVlxNXEJkqSbv7sZSk6J6tdYbnVwPWThGNiVB7Mw7bS0i3xs+q7+n9DX97lJPU3XAbw=
+    # LTD Keeper / S3 deployment credentials and configs
+    - secure: "AyDcLTZSpBsa4DCbG21TCGnU7opkm/BungRjFaQ2QZ0hV4Ht7yYX45a9qHcTvTZ/8F6isC4jsBD/YCkAB/YYq77WA6/kmoRgEUjw+J3ePsE4zk+shu2b/g+pKyhwcZpOFKb0DD1j11LrQnAtA87NYTWcJIWHup5BOpR/H7JXORUwEorabZLjBAMZ27wK3RCF4O8PdxsQV9yxpOGGJ0TjWK3XSeC82cVu+IYRBwSQ2iyjLWy7e5W5n7clt5P0f/MUAMLZ0AiZkXfer4bOlt7cxTyopCimUVt4YUM4pziu9lJy2iwtc4NYtxdsjxvT3NfuIMB+9O/Op5l1XGCmXAdHJZJ6VtL4Hpc67LBqchmLp2jTC+hj6CUY8j7WePjwtWs4lkD31NxNzUvJ3Y0tB/xG9GdRA6NkpPf1NneIfhgvobLcr6U7LmLUxLcL5MJxwHDIYs4QEv8MxG/wYUEOViPaHZRZ6Kct36JEGvD+i24++tGBFIeMxfSsicjuyMUDd7kq1e0hyfuMcOyjboUyLUNSaV/4Ur4/LYu2gXzaW7sXLZxIdBb7u/B5RPZ02F/VTZXQXXPdWGe2naHoFFrKbLtbddEMB9BuBt3I8Irrt57+073/2xjAtOBjBXYzGfjK+Vp0dedbS4ceuyHSMnAgP0R0IlbctCTCfCu9xs5sIrWtUiw="
+    - secure: "kYanZnvrMWjZoUcbV1Kxyns8ZUWhM456ej6t17C6PyeGaMbMkL4I34GjY1F6auxi8Hx2ppIWepwkcL/+fI3iZsqvn2GXjO/6yssGzkQM3IjFJNaSo1smP1t9R7l5c6WswZ2cG1qg0NwgugGxXzTo2rYPR4HVS6olGsZyzpq+6uLAp53F6JACbMMHC/DVMee6xqJ2O+YZzyX4KR3UO6dyYZtjJQjlDTTQ9Q3Z9JXDdJyn2DwBk8RFyRpDIhfzkiqFL6Yd04O/GC7fp3ZyjnvYr57KSNzv2dqRzPhhgMP1xMZ9IwlZS/81aBfLUecHDGWg3/THO+C/eEpeEjSD2mAyurg1M3HcITTcezP0260FFVIjb87tH7mHxa2mTBVflCnZrxW+PeioTD/nMwYTdY3OJ9eVBHnwb2LGeO4sAf2spV5sE7q4HTIs4msK6sMGR89QaECcZa3I++Xwaaq03oeVM3MqSuByJvOQW1Ir3ox3S3Ypwb/G2P8sQyHidmJ7ngzqtrIU3OGch88A4kYTyEN64dcHSNXwQ/gpC5X5bsFGVfOE/EhsDcBk07hPlAbJvuL+HM3vXzJ2+UulAc9luY0ESDTLJ6jKjkmsK07p5AcQP2f+m+nHnZxNewGSAqSP7DRf2NN/5VLjXkpYu5ngDXMrNee/PRkn509tpO30dVVXeQY="
```

Comment:
For PyPI that's personal credential, but I've just set up a sqre-admin PyPI account and I'll use that. We can make `sqre-admin` a *maintainer* on all our PyPI packages so it has upload privileges.

For LTD Keeper, I have a user account on `keeper.lsst.codes` that's dedicated to documentation uploads, and on has upload permissions.

For S3, I'm using a key attached to my name. So... not ideal.

Ideal solution:

- every product that uses LTD gets its own S3 credential that's scoped to only allow uploads to the specific bucket prefix of that product.
- for Keeper, we give each product it's own account so that we can rotate passwords if necessary. These accounts would also be scoped to only allow uploads for that product.

What do you think of this plan @jhoblitt? For making new AWS keys for each product, either Keeper can do that when a product is created, or maybe Vault can help out here?